### PR TITLE
Added more icons

### PR DESCRIPTION
**Pull request by tonyling94**
Added more icons to support employer and candidate chat interface.

### CODE REVIEW

**Review by ninariccimarie — CHANGES_REQUESTED** [review]
Hi @tonyling94 can you export the images from the group layer in [Figma](https://www.figma.com/file/vfbBOIGYE704NuEQkHnSlk/Sirius-Web---UI-Kit-2.0?type=design&node-id=4-30&mode=design&t=phJWSSHjl1hPJvyW-0)? you'll have to remove some sections on the svg file after as well. You can follow the instructions here: https://aries.glints.design/?path=/story/next-icon--page#adding-new-icons

File: src/@next/Icon/icons/ri-attachment-fill.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M20.997 2.992L21 21.008C20.9979 21.2706 20.8926 21.5219 20.7068 21.7075C20.521 21.8931 20.2696 21.9982 20.007 22H3.993C3.72981 22 3.47739 21.8955 3.2912 21.7095C3.105 21.5235 3.00027 21.2712 3 21.008V2.992C3.00209 2.72938 3.10742 2.47813 3.29322 2.29251C3.47902 2.1069 3.73038 2.00183 3.993 2H20.003C20.552 2 20.997 2.444 20.997 2.992ZM9 13V9C9 8.73478 9.10536 8.48043 9.29289 8.29289C9.48043 8.10536 9.73478 8 10 8C10.2652 8 10.5196 8.10536 10.7071 8.29289C10.8946 8.48043 11 8.73478 11 9V13C11 13.2652 11.1054 13.5196 11.2929 13.7071C11.4804 13.8946 11.7348 14 12 14C12.2652 14 12.5196 13.8946 12.7071 13.7071C12.8946 13.5196 13 13.2652 13 13V9C13 8.20435 12.6839 7.44129 12.1213 6.87868C11.5587 6.31607 10.7956 6 10 6C9.20435 6 8.44129 6.31607 7.87868 6.87868C7.31607 7.44129 7 8.20435 7 9V13C7 14.3261 7.52678 15.5979 8.46447 16.5355C9.40215 17.4732 10.6739 18 12 18C13.3261 18 14.5979 17.4732 15.5355 16.5355C16.4732 15.5979 17 14.3261 17 13V8H15V13C15 13.7956 14.6839 14.5587 14.1213 15.1213C13.5587 15.6839 12.7956 16 12 16C11.2044 16 10.4413 15.6839 9.87868 15.1213C9.31607 14.5587 9 13.7956 9 13Z" fill="#626262"/>
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-attachment-fill.svg]: I think you forgot to remove `fill="#626262"` part on all the svg. the indicator is they should be in black.

Comment (tonyling94) [reply src/@next/Icon/icons/ri-attachment-fill.svg]: opps, I'll push another fix. thanks!

Comment (tonyling94) [reply src/@next/Icon/icons/ri-attachment-fill.svg]: Fixed in 11488322b8202b62914b21238c37dbfbe89737f1

**Review by ninariccimarie — APPROVED** [review]
LGTM! thanks

**Pull request merged**